pkg/cache/cache.go

```diff
@@ -75,6 +75,7 @@ type Block struct {
 
 const (
 	modelIdentifier                       = "model.aibrix.ai/name"
+	nodeType                              = "ray.io/node-type"
```

Comment:
I am thinking if we want to add a aibrix label like `orchestration.aibrix.ai/server-pod`. autoscaling need that feature as well. We can start with ray specific label and replace with our own later as a separate issue

Reply:
Agree, we have implement correctly in next iteration.